scripts/download-file-proxy-map.ts

```diff
@@ -0,0 +1,46 @@
+import { writeFile } from 'node:fs/promises';
+import { buildClient } from '@datocms/cma-client-node';
+import dotenv from 'dotenv-safe';
+import { datocmsEnvironment } from '../datocms-environment';
+
+dotenv.config({
+  allowEmptyValues: Boolean(process.env.CI),
+});
+
+type FileRecord = {
+  slug?: string;
+  file: {
+    upload_id: string;
+  }
+}
+type FileSlugMap = {
+  [key: string]: string;
+};
+
+async function getFileProxyMap() {
+  // use client instead of http api for pagination support
+  const client = buildClient({
+    apiToken: process.env.DATOCMS_READONLY_API_TOKEN!,
+    environment: datocmsEnvironment,
+  });
+
+  const fileSlugMap = {} as FileSlugMap;
```

Comment:
```suggestion
  const fileSlugMap: FileSlugMap = {};
```

This is safer than `as`:

<img width="330" alt="image" src="https://github.com/user-attachments/assets/9ae66cab-6a07-4ea7-91fd-099b7ea35a3a" />